libraries/ae-inventory-processor/src/main/resources/META-INF/templates/en/inventory-statistics-vulnerability/tpc_inventory-vulnerability-statistics.dita.vt

```diff
@@ -32,19 +37,22 @@
             in the statistics with their modified severity if available or their unmodified severity otherwise.
         </p>
         <p id="statistics-table-modified">
-#statisticsOverviewTableModified("vulnerabilities_statistics_table_modified_all", "Effective Vulnerability Statistics$reportContext.inContextOf()", "", $report.getOverviewTablesVulnerabilityStatusMappingFunction())
+#statisticsOverviewTable("vulnerabilities_statistics_table_modified_all", "Context Vulnerability Statistics$reportContext.inContextOf()", $vulnerabilities, "", true)
         </p>
 
 
 ## iterate over all advisories that shall have their own modified overview table
 #foreach($advisory in $report.getGenerateOverviewTablesForAdvisories())
-        <p id="statistics-preface-modified-$advisory">
+    #set ($advisoryName = $advisory.getWellFormedName())
+    #set ($advisoryIdentifier = $advisory.name())
+##
+        <p id="statistics-preface-modified-$advisoryIdentifier">
             The following table shows statistics for the identified vulnerabilities with advisory information from
-            $advisory. The vulnerabilities are included in the statistics with their modified severity if available or
+            $advisoryName. The vulnerabilities are included in the statistics with their modified severity if available or
             their unmodified severity otherwise.
         </p>
-        <p id="statistics-table-modified-$advisory">
-#statisticsOverviewTableModified("vulnerabilities_statistics_table_modified_$advisory.toLowerCase()", "Effective Vulnerability Statistics with $advisory Advisories$reportContext.inContextOf()", "$advisory", $report.getOverviewTablesVulnerabilityStatusMappingFunction())
+        <p id="statistics-table-modified-$advisoryIdentifier">
+#statisticsOverviewTable("vulnerabilities_statistics_table_modified_$advisoryIdentifier.toLowerCase()", "Context Vulnerability Statistics with $advisoryName Advisories$reportContext.inContextOf()", $vulnerabilities, $advisory, true)
```

Comment:
Are we sure that we do not need to escape attributes directly embedded in the text.

Reply:
Do you mean the `advisoryIdentifier` / `advisoryName` in the id and title?

Reply:
All vulnerability and advisory names are now escaped when used in the id attribute or in title tags.